frontend/src/pages/org/[id]/overview/index.tsx

```diff
@@ -570,56 +577,191 @@ const OrganizationPage = withPermission(
       ws?.name?.toLowerCase().includes(searchFilter.toLowerCase())
     );
 
+    const workspacesWithFaveProp = useMemo(
```

Comment:
Same here, why not one single array of fav first and later non fav.

This would also mean the below logic is  much more simple

Reply:
For this one, I need the favorite and nonfavorite workspaces as two separate arrays because I'm using both of them separately, unlike the one in the AppLayout component.
<img width="1000" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/bd342a9f-6c18-4087-8010-da1d55d7ad5f">

I think it's more simpler this way rather than have one list as you're suggesting and then doing filtering in its usage

Reply:
Exactly, 😅 you don't need seperate list. 

What you need is a flag `isFavorite`